app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -86,8 +86,19 @@ public boolean upload(List<BgReading> glucoseDataSets, List<Calibration> meterRe
 
             if (enableMongoUpload) {
```

Comment:
Im an idiot and didnt realize why it wasnt uploading... (I thought you had hardcoded it to just upload no matter what woooops!)